### ISSUE TITLE
Dashboard is temporarily down - too many visitors!

### ISSUE THREAD

**Issue by CSSEGISandData** [CSSEGISandData/COVID-19#116]
We are looking into the issue. Sorry for the inconvenience. - JHU CSSE

**Comment by CSSEGISandData** [comment]
All services are not shared to the public now.

**Comment by AndreiD** [comment]
just proxy it with https://www.cloudflare.com/  ... they would cache some stuff...

**Comment by bjarkimg** [comment]
Feel free to use/load test MS Power BI  (updates few times a day to this datasett)
http://bit.ly/38qeMbz

![image](https://user-images.githubusercontent.com/1289419/75175672-b4b19000-572a-11ea-8059-b6bdd178d26b.png)
![image](https://user-images.githubusercontent.com/1289419/75175715-c7c46000-572a-11ea-8f3d-671a40a78faa.png)

**Comment by iceweasel1** [comment]
@CSSEGISandData Is it possible to foresee approximately when it will work again to some extent? Was there no proxy or similar protection against overload or targeted attacks before?

**Comment by joelio** [comment]
> @CSSEGISandData Is it possible to foresee approximately when it will work again to some extent? Was there no proxy or similar protection against overload or targeted attacks before?

It's running on arcgis, so I'd assume they'd have some caching in place, they might have gone over their tier allocation though

**Comment by dawenx** [comment]
Please see #7 for other tools that are currently on-line.

**Comment by iceweasel1** [comment]
> > @CSSEGISandData Is it possible to foresee approximately when it will work again to some extent? Was there no proxy or similar protection against overload or targeted attacks before?
> 
> It's running on arcgis, so I'd assume they'd have some caching in place, they might have gone over their tier allocation though

At least outside of Argic it is not running yet. Even deleting the cache etc. in the browser does not change anything. :-)

![Bildschirmfoto vom 2020-02-24 18-39-04](https://user-images.githubusercontent.com/49625256/75176829-2ee71200-5735-11ea-969c-de3802658392.png)

**Comment by joelio** [comment]
> > > @CSSEGISandData Is it possible to foresee approximately when it will work again to some extent? Was there no proxy or similar protection against overload or targeted attacks before?
> > 
> > 
> > It's running on arcgis, so I'd assume they'd have some caching in place, they might have gone over their tier allocation though
> 
> At least outside of Argic it is not running yet. Even deleting the cache etc. in the browser does not change anything. :-)
> 
> ![Bildschirmfoto vom 2020-02-24 18-39-04](https://user-images.githubusercontent.com/49625256/75176829-2ee71200-5735-11ea-969c-de3802658392.png)

Clearing your local cache won't do anything, data needs to be served from arcgis cloud and given this is so hot atm then (I assume) they've gone over limits :)

**Comment by iceweasel1** [comment]
@joelio Damn it. 😅

**Comment by jcavfen9** [comment]
Here is another one if it may help (updated only once a day with the .csv data):
https://public.tableau.com/views/Coronavirus2019-nCoVstats/Coronavirus2019-nCoV?:retry=yes&:display_count=y&:origin=viz_share_link
![image](https://user-images.githubusercontent.com/21645217/75180295-2d6d1800-573c-11ea-8d6a-1341f0deec22.png)

**Comment by duncanmak** [comment]
@bjarkimg Do you know who set up the PowerBI?

**Comment by Shane073** [comment]
@CSSEGISandData Will the dashboard be made public again?

**Comment by Shane073** [comment]
FYI, dashboard is back up now.

**Comment by bjarkimg** [comment]
> @bjarkimg Do you know who set up the PowerBI?
@duncanmak 
I did.

**Comment by CSSEGISandData** [comment]
Hello! The system is back to live!!! We appreciate your patience. We will keep monitoring the system. Thanks!

**Comment by harshil21** [comment]
> Hello! The system is back to live!!! We appreciate your patience. We will keep monitoring the system. Thanks!

Thanks most of it is working, but the cases per country tab is still showing 'not fully configured' though.

**Comment by iceweasel1** [comment]
@CSSEGISandData Since a few minutes I get the following message when I want to view/update the map. Since the malfunction was fixed yesterday, it has been running without problems until now.

![Bildschirmfoto vom 2020-02-25 16-35-13](https://user-images.githubusercontent.com/49625256/75262602-0e7a8e80-57ed-11ea-9a3f-53f993aa9996.png)

**Comment by CSSEGISandData** [comment]
> We also experienced the slow performance this morning, not with the dashboard itself, but in general ArcGIS Online. Indeed the load this morning has been very heavy. Our team is monitoring the load and actively looking into this problem. - from Esri

Hope it won't last long. Thanks!

**Comment by AZAGDAN** [comment]
Here is an alternative Dashboard created in Google Data Studio.
When original doesn't work this works pretty neatly and showing some additional analysis like bubble chart reflecting virus control effectiveness.
https://datastudio.google.com/s/p4bDfRwOuq4
![Dashboard 52](https://user-images.githubusercontent.com/61535854/75421797-5ad3e480-593b-11ea-96dd-24fe354c99ad.png)

**Comment by bsimser** [comment]
Is the .pbix available for this @bjarkimg or just the .csv files?

**Comment by harshil21** [comment]
> > Hello! The system is back to live!!! We appreciate your patience. We will keep monitoring the system. Thanks!
> 
> Thanks most of it is working, but the cases per country tab is still showing 'not fully configured' though.

Still facing this problem. Any fixes? Tried clearing cookies but didn't work. Works on mobile and browsers other than chrome.
![image](https://user-images.githubusercontent.com/37377066/75897089-723a3280-5e51-11ea-97c4-c25d9336a3dd.png)

**Comment by delmontebanana** [comment]
> Feel free to use/load test MS Power BI (updates few times a day to this datasett)
> http://bit.ly/38qeMbz
> 
> ![image](https://user-images.githubusercontent.com/1289419/75175672-b4b19000-572a-11ea-8059-b6bdd178d26b.png)
> ![image](https://user-images.githubusercontent.com/1289419/75175715-c7c46000-572a-11ea-8f3d-671a40a78faa.png)

There appears to be a 24 hour delay on the data - is there a way to manually force a refresh?

**Comment by bsimser** [comment]
Power BI reports/apps/dashboards can be put on an automated refresh but I don't see anything on here as to when the last refresh was. I'm still waiting to see if someone has the .pbix for this and will publish it, otherwise I'm making my own.

**Comment by 3verse** [comment]
Not PowerBi, but here's my contribution - if needed for backup purposes: 
[https://public.tableau.com/profile/salvatore1114#!/vizhome/wdctest/CoronavirusCOVID-19Dashboard-DailyUpdates](https://public.tableau.com/profile/salvatore1114#!/vizhome/wdctest/CoronavirusCOVID-19Dashboard-DailyUpdates)

**Comment by ast96** [comment]
I'm seeing "Not fully configured" in various panes in the dashboard now...
